### PR TITLE
Support debezium records with raw data (MongoDB records)

### PR DESCRIPTION
**Pull request by lovromazgon**
### Description

This PR contains logic to unwrap a Debezium record coming from MongoDB, i.e. a record with raw data in fields `after` and `before`.

### Quick checks:

- [X] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [X] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [X] I have written unit tests.
- [X] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: pkg/processor/procbuiltin/unwrap.go [hunk 1]
```diff
@@ -281,44 +291,60 @@ func (d *debeziumUnwrapper) validateRecord(data record.StructuredData) error {
...
+			continue // ignore
+		case debeziumFieldTimestamp:
+			tsMs, ok := val.(float64)
+			if !ok {
+				return nil, cerrors.Errorf("%s is not a float", debeziumFieldTimestamp)
+			}
+			readAt := time.UnixMilli(int64(tsMs))
+			rec.Metadata.SetReadAt(readAt)
+		case debeziumFieldSource:
+			// don't add prefix for source fields
```
Comment (hariso) [inline pkg/processor/procbuiltin/unwrap.go:309]: What's the reason for this btw?

Comment (lovromazgon) [reply pkg/processor/procbuiltin/unwrap.go:309]: For compatibility with the [SDK](https://github.com/ConduitIO/conduit-connector-sdk/blob/059da633a9feb696d0c61a27e806ed96252d1d29/record_formatter.go#L180). There we put metadata fields into `source`, so if another Conduit instance processes such a record it unwraps back the original OpenCDC record.

Comment (hariso) [reply pkg/processor/procbuiltin/unwrap.go:309]: Gotcha, thanks for the clarification! :raised_hands: Having a comment about that would be very useful IMO. :+1:

Comment (lovromazgon) [reply pkg/processor/procbuiltin/unwrap.go:309]: Added the comment here: https://github.com/ConduitIO/conduit/pull/877/commits/30d21e0a651ede80eafe04cce9f3973d971ac6ce

**Pull request merged**